source/mod-panel

```diff
@@ -456,18 +458,21 @@ if __name__ == '__main__':
     # --------------------------------------------------------------------------------------------------------
     # Set-up workdir environment as LV2_PATH, if relevant
 
-    if os.getenv("LV2_PATH") is None:
+    lv2_path = os.getenv("LV2_PATH")
+    if lv2_path is None:
         WORKDIR = os.getenv("WORKDIR", os.path.expanduser("~/mod-workdir"))
         LV2_PATH = os.path.join(WORKDIR, "x86_64", "plugins")
         if os.path.exists(LV2_PATH):
             os.environ["LV2_PATH"] = LV2_PATH
+            lv2_path = LV2_PATH
 
     # --------------------------------------------------------------------------------------------------------
     # Create and show GUI
 
     gui = PanelWindow()
     gui.show()
 
+    gui.setInformation("<b>LV2 path:</b><br/>" + "<br/>".join(lv2_path.split(":")))
```

Comment:
this call should be done before show.
better yet, pass it as argument to the class constructor.